libs/format_styled_components.js

```diff
@@ -123,4 +123,20 @@ const generateTagFormatter = ({ context, EXPECTED_NAMES, UNEXPECTED_NAMES }) =>
   }
 }
 
-module.exports = { generateTagFormatter, checkImportStyledComponents, getStyledComponentBaseName }
+const getTagName = (node) => {
+  let tag = node
+
+  if (tag.openingElement) {
+    tag = tag.openingElement
+  }
+
+  const name = tag.name
+
+  if (name?.object && name?.property) {
+    return `${name.object.name}.${name.property.name}`
+  }
+
+  return name?.name || ''
+}
```

Comment:
かなり些末ですし短い関数なので気にならないのですが、letに再代入はちょいわかりづらいかな？と思い以下のsuggestionだけ貼ります〜！

```suggestion
const getTagName = (node) => {
  const tag = node.openingElement || node
  const name = tag.name

  if (name.object && name.property) {
    return `${tagName.object.name}.${tagName.property.name}`
  }
  
  return name.name || ''
}
```

Reply:
良さそうなんで取り込みました。
ついでなんで `const name = (node.openingElement || node).name` にしちゃいますね